modules/gapi/test/gapi_transform_tests.cpp

```diff
@@ -0,0 +1,114 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019 Intel Corporation
+
+#include <tuple>
+
+#include "test_precomp.hpp"
+#include "opencv2/gapi/gtransform.hpp"
+
+namespace opencv_test
+{
+
+namespace
+{
+using GMat2 = std::tuple<GMat, GMat>;
+using GMat3 = std::tuple<GMat, GMat, GMat>;
+using GMat = cv::GMat;
+
+GAPI_TRANSFORM(my_transform, <GMat(GMat, GMat)>, "does nothing")
```

Comment:
let's add one/two for single input argument as well:
`<GMat(GMat)>` or `<GMat2(GMat)>`

Reply:
Done